spec/features/staff/users_can_add_benefitting_countries_spec.rb

```diff
@@ -1,8 +1,9 @@
 RSpec.feature "users can add benefitting countries as intended beneficiaries" do
-  context "when the user is signed as a BEIS user" do
-    let(:user) { create(:beis_user) }
+  context "when the user is signed as a delivery partner user" do
```

Comment:
Good spot!

Reply:
Yeah, I think there's a couple of tests like this, where it was just easier to be a BEIS user (no added complication around reports etc), but this feels much closer to reality.